src/codecs/md/index.ts

```diff
@@ -242,14 +244,16 @@ async function decodeRootArticle(
     yaml.safeLoad(
       typeof frontmatterYaml?.value === 'string' ? frontmatterYaml.value : ''
     ) ?? {}
-
   const context: DecodeContext = {
     frontmatter: {
       ...loadedFrontmatter,
       references: await loadExtractedReferences(loadedFrontmatter),
     },
   }
 
+  // Remove the reference to the bibliography file as it has been inlined to the `references` key by this point
+  delete context.frontmatter.bibliography
```

Comment:
What do you think about this @nokome? Should we keep the reference to the file path of the `bibliography` file, or omit it from the `meta/frontmatter` as I've done here?